lighthouse-core/scripts/i18n/pseudolocalize.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const cs = require('./correct-strings.js');
+const path = require('path');
+
+const LH_ROOT = path.join(__dirname, '../../../');
+
+cs.collectAllPreLocaleStrings(path.join(LH_ROOT, 'lighthouse-core/lib/i18n/pre-locale/'),
```

Comment:
why is this `pseudolocalize.js`? Let's just make `correct-strings.js` executable (optionally executable if called from the command line if it needs to be required anywhere else, though it doesn't look like it needs to be)